manifest.toml

```diff
@@ -35,6 +35,10 @@ ram.runtime = "250M"
     [install.domain]
     type = "domain"
 
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
```

Comment:
Won't it make this app available to visitors by default? This app is meant to be accessible to a single user (see the `install.admin` section).

Reply:
@ericgaspar ⇑

Reply:
We can remove L.38-40 and add...
```
[resources.permissions]
    main.url = "/"
    main.allowed = "admins"
```

Reply:
Tested that and sounds good. I push that change.